ui/src/main.mjs

```diff
@@ -257,10 +269,39 @@ fetch('_setup')
         app.mount('#app')
     })
     .catch((err) => {
```

Comment:
@joepavitt Removing console.log(err) resulted in a linting error on this line... We could just remove `err` to fix it but my thoughts are we keep caught error message for reference and further processing later. Do you approve of this addition?:  
```js
error = { error: err, type: 'server unreachable', message: 'There was an error loading the Dashboard.' }
```